dash-pipeline/bmv2/dash_outbound.p4

```diff
@@ -9,19 +9,32 @@
 control outbound(inout headers_t hdr,
                  inout metadata_t meta)
 {
-    action route_vnet(bit<16> dst_vnet_id) {
+    action set_route_meter_attrs(bit<1> meter_policy_en,
+                                 bit<16> meter_class) {
+        meta.meter_policy_en = meter_policy_en;
+        meta.meter_class = meter_class;
```

Comment:
did you mean to assign meter_class here or route_meter_class? and then later use it in the condition below where you pick policy/route/mapping meter_class?

Reply:
yes, that is right. fixed this. thnx!